src/components/checkbox/checkbox.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Checkbox from './checkbox';
+
+export default {
+  component: Checkbox,
+  title: 'Components/Checkbox',
+  parameters: {
+    layout: 'fullscreen',
+  },
```

Comment:
This leads to no padding from Storybook. I suggest to only use this for full-screen elements that span the entire screen width. For smaller inline elements like this I suggest we don't use the `fullscreen` layout.
```suggestion
```